apps/website/src/examples/Input.button.tsx

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { LabeledInput, IconButton } from '@itwin/itwinui-react';
+import { SvgCloseSmall } from '@itwin/itwinui-icons-react';
+
+export default () => {
+  return (
+    <>
+      <LabeledInput
+        label='Address'
+        placeholder='1600 Pennsylvania Avenue, Washington, DC 20500'
+        defaultValue='685 Stockton Dr, Exton, PA 19341'
+        svgIcon={
+          <IconButton styleType='borderless'>
```

Comment:
icon buttons should have an [accessible name](https://www.tpgi.com/what-is-an-accessible-name/), which you can specify using `aria-label`

Reply:
Added an `aria-label` to the button example.